s3fs/tests/test_s3fs.py

```diff
@@ -2070,9 +2071,13 @@ def test_via_fsspec(s3):
     import fsspec
 
     s3.mkdir("mine")
-    with fsspec.open("mine/oi", "wb") as f:
+    with fsspec.open(
+        "s3://mine/oi", "wb", client_kwargs={"endpoint_url": endpoint_uri}
+    ) as f:
         f.write(b"hello")
-    with fsspec.open("mine/oi", "rb") as f:
+    with fsspec.open(
+        "s3://mine/oi", "rb", client_kwargs={"endpoint_url": endpoint_uri}
+    ) as f:
```

Comment:
This test is unrelated to glob double asterisk but I think it was wrong. It tested the `LocalFileSystem` implementation instead of the S3 one.